File: RA1/clase 14/JAVA/AbstractClasses/Gato.java

public class Gato extends Animal{
    private int flexibilidad;

    public Gato(String nombre, String especie, String raza, int flexibilidad){
        this.nombre = nombre;
        this.especie = especie;
        this.raza = raza;
        this.flexibilidad = flexibilidad;
    }

    public void haceSonido(){
        System.out.println(this.nombre + " maulla....");
    }
}

File: RA1/clase 14/PYTHON/Gato.py
from Animal import Animal

class Gato(Animal):
    def __init__(self, nombre, especie, raza, flexibilidad):
        self.nombre = nombre
        self.especie = especie
        self.raza = raza
        self.flexibilidad = flexibilidad

    def hace_sonido(self):
        print(self.nombre + " esta maullando...")
